src/components/pagination/pagination.tsx

```diff
@@ -4,7 +4,13 @@ import { Pagination, ThemeProvider } from '@mui/material';
 import { useLocation, useNavigate } from 'react-router-dom';
 import styles from './styles';
 
-const BasicPagination = ({actualPage, query, countPages = 10}) => {
+type TBasicPagination = {
+  actualPage: string | null;
+  query?: string | null;
```

Comment:
? means optional, so null is redundant here

query?: string;
actualPage?: string;